src/Tracy/Debugger.php

```diff
@@ -470,6 +470,13 @@ public static function timer($name = NULL)
 		$now = microtime(TRUE);
 		$delta = isset($time[$name]) ? $now - $time[$name] : 0;
 		$time[$name] = $now;
+		if (!self::$productionMode) {
+			static $panel;
+			if (!$panel) {
+				$panel = self::getBar()->getPanel('time');
+			}
+			$panel->data[] = array('name' => $name, 'time' => $now, 'delta' => $delta);
+		}
```

Comment:
> The information about timestamp of timer "event" and the time since last timer usage is the advantage.

Ok, you convinced me. Keep the `$now` info. We we'll see how useful it is in practice.

> The performance improvement 

Improvement? I made it intentionally slower by removing your performance improvement (static cache for `$panel`). I agree that **your** improvement was super micro optimization, so I removed it.